permissions/plugin-vrealize-automation-8.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "vrealize-automation-8-plugin"
```

Comment:
Name doesn't match artifact ID, so this is the reason the upload fails.

Reply:
Thanks! I knew it was something stupid obvious! :)

Reply:
It would be nice if the automatic checker would catch that.

Reply:
@prydin the automatic checker will create the PR in the future so this shouldn't be an issue, but currently the checker only checks the hosting ticket since this file is not created prior to the hosting process completing.

Reply:
The PR build could still fail like it does for users that don't exist yet; it's just that so far nobody's cared enough yet to implement it.

Reply:
Before I submit a new PR, the fix is to remove "-plugin" from the name, right?

Reply:
@prydin  Yes, it should match the artifactId from your pom.xml